www/contribute/contribute_guidelines.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: contribute-help
+title: Apache Cordova Contribute Guidelines
+---
+# Contributor Guidelines
+
+Thanks for helping to improve Cordova! This page provides a general guide on making contributions to Cordova. If you can't find something on this page, please feel free to contact the [dev mailing list](http://cordova.apache.org/contact) or ask questions on the Cordova Slack.
+
+## Prerequisites
+
+Before contributing to Apache Cordova, please complete the following steps:
+
+1. Sign the [Individual Contributor License Agreement (ICLA)](http://www.apache.org/licenses/#clas) and [submit it to the ASF](http://www.apache.org/licenses/#submitting).
+    * You should receive an email confirming you submission within a day or so and your name will appear in the [list of committers and non-committers](https://people.apache.org/committer-index.html)
+    * If you are submitting on behalf of a company, you may need to submit a CCLA as well
+2. Join the [mailing list](http://cordova.apache.org/contact/) and send a brief introduction of yourself
+3. Create an [Apache JIRA](https://issues.apache.org/jira/secure/Signup!default.jspa) account
+
+## Working with JIRA
+
+Issues for Apache Cordova are hosted in the Apache JIRA. All code contributions made to Cordova should have a corresponding JIRA issue. When reporting issues, please follow [these guidelines](./issues.html).
+
+#### Claiming Issues
+If you find a JIRA issue that you would like to work on, you can ask to claim it; please leave a comment indicating your intention and a committer will assign it to you. Some issues in JIRA are auto-assigned to certain contributors. If it is clear that an issue is not being worked on, feel free to work on it yourself (but please comment first to let the asignee know). If you are looking for a place to start, try searching the [issues labelled easyfix](https://issues.apache.org/jira/browse/CB-9974?jql=project%20%3D%20CB%20AND%20resolution%20%3D%20Unresolved%20AND%20labels%20%3D%20%22easyfix%22%20ORDER%20BY%20createdDate%20DESC).
+
+
+## Submitting Code

```

Comment:
Do you want to add some information from the coho repo here?
https://github.com/apache/cordova-coho/blob/master/docs/committer-workflow.md

Some steps are useful, specially for new folks using git

We also have a one for submitting code for review, which in summary is use Github and mail dev list
https://github.com/apache/cordova-coho/blob/master/docs/code-reviews.md